### PR TITLE
Stop publishing nightly builds of VSIX packages

### PR DESCRIPTION
**Pull request by JoeRobich**
Removes VSIX publishing configuration and support from our nightly builds.

### CODE REVIEW

**Comment by RikkiGibson** [comment]
Why do we want to make this change?

**Comment by JoeRobich** [comment]
> Why do we want to make this change?

We have moved from myget to AzDO Artifacts. At the moment there are few (if any) downloads of our VSIXs from myget.

**Review by RikkiGibson — APPROVED** [review]

**Comment by RikkiGibson** [comment]
If this helps ease the transition, that's fine. I'm guessing our AzDo publishing is managed by some AzDo task and not by our in-repo scripts..?

**Comment by jasonmalinowski** [comment]
@JoeRobich Are we able to delete the support for VSIX pubishing too?

**Comment by JoeRobich** [comment]
> Are we able to delete the support for VSIX pubishing too?

Sure. I will update.

File: eng/publish-assets.ps1 [hunk 1]
```diff
@@ -85,38 +85,18 @@ function Publish-Nuget($publishData, [string]$packageDir) {
...
-    
-    Write-Host "  Publishing '$vsix'"
-    if (-not $test) { 
-      $response = Invoke-WebRequest -Uri $uploadUrl -Headers @{"X-NuGet-ApiKey"=$apiKey} -ContentType 'multipart/form-data' -InFile $vsix -Method Post -UseBasicParsing
-      if ($response.StatusCode -ne 201) {
-        throw "Failed to upload VSIX extension: $vsix. Upload failed with Status code: $response.StatusCode"
-      }
-    }
-  }
-}
```
Comment (jaredpar) [inline eng/publish-assets.ps1:106]: Good riddance to code I wrote so many years ago.

**Review by jaredpar — APPROVED** [review]

**Pull request merged**